lib/moleculer/support/hash.rb

```diff
@@ -19,13 +47,17 @@ module HashUtil
       # @return [Object] the value at the given key
       def fetch(hash, key, default = :__no_default__)
         return fetch_with_string(hash, key, default) if key.is_a?(String) || key.is_a?(Symbol)
-        return hash.fetch(key, default) if default != :__no_default__
+        return Support::Hash.fetch(key, default) if default != :__no_default__
```

Comment:
if we are going to have a double guard clause then we should align the conditions.